pkg/dcgmexporter/system_info.go

```diff
@@ -146,6 +155,15 @@ func SwitchIdExists(sysInfo *SystemInfo, switchId int) bool {
 	return false
 }
 
+func CPUIdExists(sysInfo *SystemInfo, cpuId int) bool {
+	for _, cpu := range sysInfo.CPUs {
+		if cpu.EntityId == uint(cpuId) {
+			return true
+		}
+	}
+	return false
+}
+
 func GpuInstanceIdExists(sysInfo *SystemInfo, gpuInstanceId int) bool {
```

Comment:
Because we renamed `Cpu` to `CPU`, please rename `Gpu` to `GPU`.

Reply:
I would prefer to do that work in a separate PR since that code was not originally touched in this patch.